ironfist/src/cpp/adventure/adv.h

```diff
@@ -42,9 +42,9 @@ enum SECONDARY_SKILL
   SECONDARY_SKILL_ESTATES = 13,
 };
 
-#define NUM_RESOURCES 7
```

Comment:
This is some weird end line format difference that I couldn't unstage, Same for things below